Assets/Thirdweb/Core/Scripts/Wallets/ThirdwebLocalHdWallet.cs

```diff
@@ -0,0 +1,90 @@
+using System.Numerics;
+using System.Threading.Tasks;
+using Nethereum.Web3;
+using Nethereum.Web3.Accounts;
+
+namespace Thirdweb.Wallets
+{
+    public class ThirdwebLocalHdWallet: IThirdwebWallet
+    {
+        private Account _account;
+        private Web3 _web3;
+        private readonly WalletProvider _provider;
+        private readonly WalletProvider _signerProvider;
+
+        public ThirdwebLocalHdWallet()
+        {
+            _account = null;
+            _web3 = null;
+            // I guess, we can use the local wallet provider here to avoid breaking conditions that rely on the local wallet provider
+            _provider = WalletProvider.LocalWallet;
+            _signerProvider = WalletProvider.LocalWallet;
+        }
+
+        public Task<string> Connect(WalletConnection walletConnection, string rpc)
+        {
+            // For now we're setting the seed password to null, however in the future we may want to allow the user to set a password along with their mnemonic
+            // Also, we're only using the first account in the wallet, but in the future we may want to allow the user to select which account they want to use
+            _account = new Nethereum.HdWallet.Wallet(walletConnection.mnemonic, null).GetAccount(0);
```

Comment:
What happens if no mnemonic is passed? Should be handled here and likely randomly generated?
Also the password can be walletConnection.password

Reply:
If no mnemonic is passed, the user will get an `ArgumentNullException: Value cannot be null.` error. in the case of the empty string:  `NotSupportedException: Unknown`. Theoretically, we could randomly generate a mnemonic and then print it to the console, but I'm not sure that's the best option. If we want to pass `walletConnection.password` as the seed password, I strongly suggest we document it well. I had such a case where, in setting the password for the wallet, I passed the user's password to this option, and as a result, even if the user provided the correct mnemonic but a different password, an entirely new wallet would be created.

Reply:
I was thinking more for the situation where you just want to generate an HD Wallet, if you were building a wallet app that might be useful, specially in the future when we support multiple accounts.
As for password right now with `LocalWallet` if you don't pass it we use `SystemInfo.deviceUniqueIdentifier`, but either way works for me, not supporting it is fine in this context as well.